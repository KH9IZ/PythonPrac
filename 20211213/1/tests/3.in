[92, 80, 84, 97, 3, 35, 92, 55, 68, 57, 68, 62, 83, 26, 33, 83]
