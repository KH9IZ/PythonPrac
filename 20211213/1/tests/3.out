[3, 26, 33, 35, 55, 57, 62, 68, 68, 80, 83, 83, 84, 92, 92, 97]
